src/constants/index.ts

```diff
@@ -15,8 +15,9 @@ export const URL_PARAM_BASE_TYPES = "base_types.json";
 export const URL_PARAM_CUSTOM_TYPES = "custom-types";
 // URLs
 export const BASE_API_URL = `/api/${API_VERSION}`;
+// todo octopus launch: this is pointing to the staging octopus http endpoint
 export const DATA_BUCKET_URL =
-    "https://aics-simularium-data.s3.us-east-2.amazonaws.com";
+    "https://staging-simularium-ecs.allencell.org:443/download";
```

Comment:
Since we're handling download requests through octopus now, we should just use the `BACKEND_SERVER_IP`, like we do for defining the net connection settings, and remove this constant. Hard coding it to look at staging like we have here would break in production, because there won't be the same files in each for the case of autoconversion

Reply:
Ok, fixed this! We talked abou this, but I'm posting here for context for other reviewers that the `.simularium` file extension needed to be appended to the `fileId` earlier in the process to ensure that the download links for converted and other networked files also looked the same.